lib/eventFactory.js

```diff
@@ -265,6 +265,48 @@ function getLatestWorkflowGraph(config) {
     return Promise.resolve(pipeline.workflowGraph);
 }
 
+/**
+ * Update the workflowGraph
+ * @method updateWorkflowGraph
+ * @param  {Object}         config.*
+ * @param  {Pipeline}       pipeline.*          Pipeline
+ * @param  {Number}         pipeline.id         Pipeline id
+ * @param  {Object}         eventConfig.*
+ * @param  {Number}         [eventConfig.prNum] PR number
+ * @param  {Object}         workflowGraph       WorkflowGraph
+ * @resolves {Object}                           Resolves with workflowGraph
+ */
+function updateWorkflowGraph(config) {
+    const { pipeline, eventConfig, workflowGraph } = config;
+
+    if (pipeline.prChain) {
```

Comment:
Don't you need to check `eventConfig.prRef` here?
Only event PR has prChain?

Reply:
No, we must check the base branch (`pipeline` object)'s `prChain` flag.
Please see https://github.com/screwdriver-cd/models/pull/332

Reply:
When the event come from commit, is there possibility the `pipeline.prChain` has true? 
If so maybe you also need to check the event is came from pr.
Then the if sentense will be `if (eventConfig.prRef && pipline.prChain)`.

Reply:
I agree with `if (eventConfig.prRef && pipline.prChain)` .
We need check if the event is pr or not.

Reply:
@yuichi10 Your advice is very correct. I fixed it. Thanks!